src/brouwers/kits/static/kits/js/modelkit.widget.js

```diff
@@ -320,13 +320,13 @@ function initTypeaheads() {
 
     let fields = conf.typeahead;
 
-    for (let f in fields) {
+    for (let f of Object.keys(fields)) {
         let fieldConfig = fields[f];
         let _baseSelector = `#id_${ conf.prefix_add }-${ f }`;
         let hiddenInput = $(_baseSelector);
         let input = $(`${ _baseSelector }_ta`);
 
-        let sanitize = fieldConfig.sanitize || function(query) {return query};
+        let sanitize = fieldConfig.sanitize || function(query) { return query };
```

Comment:
this *might* actually be a bit cleaner like this: `let sanitize = fieldConfig.sanitize || query => query;`, if it works :P